src/AdminConsole/Db/AddDatabaseExtensionMethod.cs

```diff
@@ -12,26 +18,43 @@ public static IServiceCollection AddDatabase(this IServiceCollection services, W
         var migrating = builder.Configuration.GetValue<string>("ef_migrate");
         if (migrating == "1")
         {
-            services.AddDbContext<MssqlConsoleDbContext>();
-            services.AddDbContext<SqliteConsoleDbContext>();
+            services.AddDbContextFactory<MssqlConsoleDbContext>();
+            services.AddDbContextFactory<SqliteConsoleDbContext>();
```

Comment:
Question: why are we not using `AddDatabaseContext(...)` (your method) here too?

Reply:
Because then it would resolve DataService<SqliteConsoleDbContext> and DataService<MsSqlConsoleDbContext> to IDataService. That would not go well.

I don't think this part is even still being used.